com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueProcessor.cs

```diff
@@ -58,7 +58,7 @@ public void ProcessReceiveQueue(NetworkUpdateStage currentStage, bool isTesting)
 
                         if (!isTesting)
                         {
-                            NetworkManager.InvokeRpc(currentQueueItem);
+                            NetworkManager.Singleton.InvokeRpc(currentQueueItem);
```

Comment:
```suggestion
                            NetworkManager.InvokeRpc(currentQueueItem);
```

Reply:
Still not following how this would be valid

Reply:
`NetworkManager.InvokeRpc` is a static function.